### DISCUSSION TITLE
Duda respecto a los tests en la T0

### DISCUSSION THREAD

**Discussion by BenjaCarrill0** [IIC2133-PUC/2024-1#11] [Tarea 0] [answered]
Buenas tardes ¿los tests para probar el código uno mismo los debe crear o deberían estar ya en el repositorio? Lo pregunto porque en caso de ser la segunda opción no los tengo :(

**Comment by CataAlamos** [comment]
Hola! Los tests los vamos a liberar luego de un par de días.

**Reply by BenjaCarrill0** [reply]
Graciass

**Comment by PabloNavarroY** [comment, marked as answer]
Puedes encontrar el link a los tests en el anuncio de la tarea o [acá](https://drive.google.com/drive/folders/1SZAYrsW6QzvnxJYK_sqfxS8ZDS6qR7IR).

**Reply by RodolfoKompatzki** [reply]
Hola! Hay algunos test cases que vienen con un nombre terminado en "-inc", que significa eso? Ya que obtengo resultados incorrectos particularmente en esos.
